lib/ansible/module_utils/cloudfront.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
```

Comment:
will do this as a final thing as am afraid I'll break something in my branch doing a pull from devel (being somewhat of a a git n00b and all)